addon/utils/bugsnag.js

```diff
@@ -0,0 +1,7 @@
+export function getMetaData(/* error, container */) {
+  return {};
+}
+
+export function getUser(/* container */) {
```

Comment:
I opted to keep both overridable methods in this single util. It seems like a cleaner interface for consumers so they don't have to continually generate `app/utils/bugsnag-*` utils every time this addon exposes more options.